modules/fedimint-ln-server/src/lib.rs

```diff
@@ -85,7 +85,19 @@ lazy_static! {
         "contracts::FundedContract::Outgoing"
     ))
     .unwrap();
-    static ref AMOUNTS_BUCKETS_SATS: Vec<f64> = vec![0.0, 0.5, 1.0, 1000.0];
+    static ref AMOUNTS_BUCKETS_SATS: Vec<f64> = vec![
+        0.0,
+        0.5,
```

Comment:
0.1 for consistency? :D

Reply:
nops

Reply:
I don't get it. Is there something specially meaningful about 0.5 (500msat)? :D

BTW. I would consider `0.1 0.5 1 5 10 50` anyway. 10x granularity is kind of a big step. Typically I'd see powers of 2, or `1, 2, 5, 10` for stuff like that.

Reply:
I chose 0.5 a long time ago, but you're right, 0.1 is also fine for these purposes